src/Geometry.h

```diff
@@ -34,9 +34,11 @@ struct fsr_data {
   /** The FSR ID */
   int _fsr_id;
 
-  /** Characteristic point in Universe 0 that lies in FSR */
+  /** Characteristic point in Root Universe that lies in FSR */
   Point* _point;
```

Comment:
I thought about that, but there are cases where the centroid can be outside of the FSR (e.g. a ring with no sectors) so it is necessary to have a point that lies within the FSR to construct the linked list of coords that uniquely identifies the FSR. 

I'll add a destructor for the point and centroid.